apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -221,6 +229,15 @@ export default function RowColumn({ index, style, data, column = false }) {
       <Grid item style={cellStyle} className={classes.cell} title={`${label}`}>
         {ranges.length === 0 ? getField({ lbl: label, color: 'inherit' }) : getFieldWithRanges({ lbls: labels })}
       </Grid>
+
+      {isSelected && fequencyText && (
```

Comment:
When looking in the UI drawings for analytic creation the frequency is displayed regardless if the checkbox selected or not. Are there requirements from other use cases to only display frequency when the checkbox is selected?

Reply:
All rows should have a frequency count, also non-selected. Non-selected values will get a frequency count of null, NaN or something like that, which should visually translate to a dash `-`.

Compare to QCS stage (https://qcs.us.qlik-stage.com/):

![image](https://user-images.githubusercontent.com/5780544/155522995-3f7483d3-86d7-4c7b-906c-4713b304d143.png)

Reply:
Good catch! :) Fixed in a30463c84f6c13c30033828747238529a00443fd